templates/telco/vnf.html

```diff
@@ -0,0 +1,459 @@
+{% extends "templates/one-column.html" %}
+
+{% block title %}VNF | Openstack{% endblock %}
+
+{% block meta_description %}Virtual Network Functions infrastructure with Charmed OpenStack{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1xq3tFh1vWhNm1gFzddB6LQ65DggVMHsRUSXvqFPqYc0/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip is-bordered u-no-padding--top u-no-padding--bottom">
+  <div class="p-strip--suru-topped">
+    <div class="row u-equal-height u-vertically-center">
+      <div class="col-7">
+        <h1>
+          OpenStack for NFV
+        </h1>
+        <p class="p-heading--3">
+          Carrier-grade infrastructure for your VNFs
+        </p>
+        <p>
+          Through Charmed OpenStack, you can have an OpenStack distribution that’s deployable, maintainable and upgradable economically. This is achieved by putting full automation around OpenStack deployments and exposed deployment operations.
+        </p>
+        <p>
+          <a href="/telco/contact-us" class="p-button--positive js-invoke-modal">Contact us about NFV</a>
```

Comment:
This contact us is not going anywhere and it's not opening the interactive form either.

When you click on it, it opens up a 404 page at the bottom

Reply:
Forgot to add in the description that I am waiting for the main telco page to be merged so we can test it. I will add a blocked label to this for now.